### DISCUSSION TITLE
Connected search webparts and multi-value strings

### DISCUSSION THREAD

**Discussion by awdhaan** [microsoft-search/pnp-modern-search#3267] [Q&A]
I'm working on a solution using document sets for dossier management. Each document set uses the unique document id. Common usage is that one document set acts as the parent and other document sets are related. So the parent has an additional site column (multiline text/note) called "Related dossiers" that contains a comma-separated list of document id's. I'm trying to find the parent document set using the unique document set through one search result web part where a second result web part should show the related document sets based on the value of the "Related dossier" site column (which is mapped to managed property RefinableStringxxx). I'm able to show expected results in the second web part if the "Related dossiers" only contains 1 document Id.
How do I make this work when the "Related dossiers" site column contains multiple document id's? Let's say I enter 2, then I would expect 2 related document sets in the second results web part. Is this something that requires and extension? Or can it be accomplished with the built-in capabilities? Thanks for any pointers or help. Love the web parts. They're Simply awesome!

**Comment by wobba** [comment]
Seems we need some string manipulatin capability for this case as I assume the key you pass over is a delimited string which has to broken into an OR query.
